runtime/doc/editorconfig.txt

```diff
@@ -86,4 +40,48 @@ Example: >lua
       vim.b[bufnr].foo = val
     end
 <
- vim:tw=78:ts=8:et:sw=4:ft=help:norl:
+
+                                                     *editorconfig-properties*
+
+The following properties are supported by default:
+
+
+charset                                                 *editorconfig.charset*
```

Comment:
Not for this PR, but it would be a good idea of matching `.contains` checks to their respective documentation. This kind of "foo can only be one of [...] types` can easily (and unintentionally) become outdated.

Reply:
Sorry I don't understand. Did you mean to comment in a different line. (I'm on mobile)

Reply:
Nope, this is the right line.

What I meant is that there are a lot of functions that have something like "`foo` should be one of `a`, `b` or `c`", and the function has some kind of check with `vim.validate`. I'm just pointing out how the documentation of those functions might get out of sync with the actual valid inputs.

But after writing this down I realize that it will be hard to add a generated check for those because it would depend on the function implementation, so ignore me lol.